From: "Matteo F. Vescovi" <m.vescovi@revese.it>
Date: Wed, 14 Dec 2011 09:11:51 +0100
Subject: locales_directory_install

---
 source/creator/CMakeLists.txt |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 62cc7be..0edc5f7 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -350,9 +350,9 @@ if(UNIX AND NOT APPLE)
 	if(WITH_INTERNATIONAL)
 		install(
 			DIRECTORY
-				${CMAKE_SOURCE_DIR}/release/bin/.blender/locale
+				${CMAKE_SOURCE_DIR}/release/bin/.blender/locale/
 				${CMAKE_SOURCE_DIR}/release/bin/.blender/fonts
-			DESTINATION ${TARGETDIR_VER}/datafiles
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/locale
 			PATTERN ".svn" EXCLUDE
 		)
 	endif()
@@ -422,9 +422,9 @@ elseif(WIN32)
 	if(WITH_INTERNATIONAL) # same as linux!, deduplicate
 		install(
 			DIRECTORY
-				${CMAKE_SOURCE_DIR}/release/bin/.blender/locale
+				${CMAKE_SOURCE_DIR}/release/bin/.blender/locale/
 				${CMAKE_SOURCE_DIR}/release/bin/.blender/fonts
-			DESTINATION ${TARGETDIR_VER}/datafiles
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/locale
 			PATTERN ".svn" EXCLUDE
 		)
 
@@ -646,9 +646,9 @@ elseif(APPLE)
 	if(WITH_INTERNATIONAL)
 		install(
 			DIRECTORY
-				${CMAKE_SOURCE_DIR}/release/bin/.blender/locale
+				${CMAKE_SOURCE_DIR}/release/bin/.blender/locale/
 				${CMAKE_SOURCE_DIR}/release/bin/.blender/fonts
-			DESTINATION ${TARGETDIR_VER}/datafiles
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/locale
 			PATTERN ".svn" EXCLUDE
 		)
 	endif()
-- 
