Author: Sergey Sharybin <sergey@blender.org>
Origin: upstream, https://developer.blender.org/rB0d4e519b748c
Bug-Debian: https://bugs.debian.org/876490
Reviewed-By: Matteo F. Vescovi <mfv@debian.org>
Last-Update: 2017-09-25

--- blender-2.79+dfsg0.orig/intern/openvdb/intern/openvdb_writer.cc
+++ blender-2.79+dfsg0/intern/openvdb/intern/openvdb_writer.cc
@@ -45,7 +45,11 @@ void OpenVDBWriter::insert(const openvdb
 
 void OpenVDBWriter::insert(const openvdb::GridBase &grid)
 {
+#if (OPENVDB_LIBRARY_MAJOR_VERSION_NUMBER == 3)
 	m_grids->push_back(grid.copyGrid());
+#else
+	m_grids->push_back(grid.copyGridWithNewTree());
+#endif
 }
 
 void OpenVDBWriter::insertFloatMeta(const openvdb::Name &name, const float value)
