From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
Date: Wed, 4 Apr 2012 10:53:23 +0200
Subject: Fix_german_translation_of_desktop_file

---
 release/freedesktop/blender.desktop |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/release/freedesktop/blender.desktop b/release/freedesktop/blender.desktop
index 34b2b9b..be142c0 100644
--- a/release/freedesktop/blender.desktop
+++ b/release/freedesktop/blender.desktop
@@ -2,10 +2,11 @@
 Name=Blender
 GenericName=3D modeller
 GenericName[es]=modelador 3D
-GenericName[de]=3D Modellierer
+GenericName[de]=3D-Modellierer
 GenericName[fr]=modeleur 3D
 GenericName[ru]=Редактор 3D-моделей
 Comment=3D modeling, animation, rendering and post-production
+Comment[de]=3D-Modellierung, Animation, Rendering und Nachbearbeitung
 Comment[es]=modelado 3D, animación, renderizado y post-producción
 Exec=blender
 Icon=blender
