From: Pino Toscano <pino@debian.org>
Date: Wed, 26 Jun 2013 11:44:00 +0200
Subject: fix_FTBFS_on_non-linux_architectures

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0a58232..e4e6518 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -896,6 +896,9 @@ if(UNIX AND NOT APPLE)
 	endif()
 
 	set(PLATFORM_LINKFLAGS "${PLATFORM_LINKFLAGS} -pthread")
+	if(CMAKE_DL_LIBS)
+			list(APPEND PLATFORM_LINKLIBS ${CMAKE_DL_LIBS})
+	endif()
 
 	# lfs on glibc, all compilers should use
 	add_definitions(-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE)
