From: "Matteo F. Vescovi" <mfv@debian.org>
Date: Mon, 25 Jul 2016 16:58:26 +0200
Subject: fix_manpage_creation

---
 doc/manpage/blender.1.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/manpage/blender.1.py b/doc/manpage/blender.1.py
index cdd2d7c..5285671 100755
--- a/doc/manpage/blender.1.py
+++ b/doc/manpage/blender.1.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # ##### BEGIN GPL LICENSE BLOCK #####
 #
