From: "Matteo F. Vescovi" <mfvescovi@gmail.com>
Date: Mon, 24 Mar 2014 08:48:07 +0100
Subject: fix_FTBFS_on_unofficial_64bit_archs

---
 intern/atomic/atomic_ops.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/intern/atomic/atomic_ops.h b/intern/atomic/atomic_ops.h
index b5a73b1..bb55ac1 100644
--- a/intern/atomic/atomic_ops.h
+++ b/intern/atomic/atomic_ops.h
@@ -61,7 +61,7 @@
 #  endif
 #endif
 
-#if defined(_M_X64) || defined(__amd64__) || defined(__x86_64__)
+#if defined(_M_X64) || defined(__amd64__) || defined(__x86_64__) || defined(__s390x__) || defined(__powerpc64__) || defined(__aarch64__) || (defined(__sparc__) && defined(__arch64__))
 #  define LG_SIZEOF_PTR 3
 #  define LG_SIZEOF_INT 2
 #else
