From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
Date: Tue, 24 Apr 2012 10:57:44 +0200
Subject: Fix_FTBFS_on_openjpeg_transition

---
Author: Mathieu Malaterre <malat@debian.org>
Bug-Debian: http://bugs.debian.org/670219
Forwarded: no
Reviewed-By: Matteo F. Vescovi <mfv.debian@gmail.com>
Last-Update: 2012-04-24

 source/blender/imbuf/intern/jp2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/blender/imbuf/intern/jp2.c b/source/blender/imbuf/intern/jp2.c
index f8082c9..2e7a65a 100644
--- a/source/blender/imbuf/intern/jp2.c
+++ b/source/blender/imbuf/intern/jp2.c
@@ -329,7 +329,7 @@ static int initialise_4K_poc(opj_poc_t *POC, int numres)
 
 static void cinema_parameters(opj_cparameters_t *parameters)
 {
-	parameters->tile_size_on = false;
+	parameters->tile_size_on = OPJ_FALSE;
 	parameters->cp_tdx=1;
 	parameters->cp_tdy=1;
 	
