From: Sergey Sharybin <sergey.vfx@gmail.com>
Date: Thu, 12 Dec 2013 14:41:06 +0100
Subject: fix_ftbfs_on_armel

---
 intern/atomic/atomic_ops.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/intern/atomic/atomic_ops.h b/intern/atomic/atomic_ops.h
index 243e194..b5a73b1 100644
--- a/intern/atomic/atomic_ops.h
+++ b/intern/atomic/atomic_ops.h
@@ -35,10 +35,19 @@
 #  include <libkern/OSAtomic.h>
 #elif defined(_MSC_VER)
 #  include <windows.h>
+#elif defined(__arm__)
+/* Attempt to fix compilation error on Debian armel kernel.
+ * arm7 architecture does have both 32 and 64bit atomics, however
+ * it's gcc doesn't have __GCC_HAVE_SYNC_COMPARE_AND_SWAP_n defined.
+ */
+#  define JE_FORCE_SYNC_COMPARE_AND_SWAP_8
+#  define JE_FORCE_SYNC_COMPARE_AND_SWAP_4
 #endif
 
 /* needed for int types */
 #include "../../source/blender/blenlib/BLI_sys_types.h"
+#include <stdlib.h>
+#include <stddef.h>
 
 /* little macro so inline keyword works */
 #if defined(_MSC_VER)
