From: Kevin Roy <kiniou@gmail.com>
Date: Thu, 26 Jan 2012 23:56:40 +0100
Subject: fix_implicit_declaration_of_guardealloc

---
 source/blender/avi/intern/endian.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/source/blender/avi/intern/endian.c b/source/blender/avi/intern/endian.c
index 1948806..94734b0 100644
--- a/source/blender/avi/intern/endian.c
+++ b/source/blender/avi/intern/endian.c
@@ -43,6 +43,10 @@
 #include "avi_intern.h"
 
 #ifdef __BIG_ENDIAN__
+#include "MEM_guardedalloc.h"
+#endif
+
+#ifdef __BIG_ENDIAN__
 static void invert (int *num) {
 	int new=0,i,j;
 
-- 
